src/core/Akka/Actor/CoordinatedShutdown.cs

```diff
@@ -534,7 +534,7 @@ internal static void InitPhaseActorSystemTerminate(ActorSystem system, Config co
                         // We must spawn a separate Task to not block current thread,
                         // since that would have blocked the shutdown of the ActorSystem.
                         var timeout = coord.Timeout(PhaseActorSystemTerminate);
-                        return Task.Run(() =>
+                        Task.Run(() =>
```

Comment:
Not supposed to return a `Task` here. Supposed to just let the process run asynchronously and invoke once the `ActorSystem` has finished terminating.